sys-fs/lvm2/lvm2-2.03.14-r4.ebuild

```diff
@@ -13,8 +14,8 @@ SRC_URI="ftp://sourceware.org/pub/lvm2/${PN/lvm/LVM}.${PV}.tgz
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux"
-IUSE="readline static static-libs systemd lvm2create-initrd sanlock selinux +udev +thin device-mapper-only"
-REQUIRED_USE="device-mapper-only? ( !lvm2create-initrd !sanlock !thin )
+IUSE="readline static static-libs systemd lvm lvm2create-initrd sanlock selinux +udev +thin"
```

Comment:
I feel like we need a news item for this if we're not going to default it on (and I don't want to default it on)

Reply:
better idea: let's `+lvm` for now, and then prepare the news item. Doing news item now means this PR has to wait another X weeks before we can actually change the negative logic to positive